### PR TITLE
[11/n] Add vscode editor bundle

### PR DESCRIPTION
**Pull request by saqadri**
[11/n] Add vscode editor bundle

Save main.js bundle representing the vscode editor webview inside the extension itself.

This will allow us to publish the extension and it can use the bundle directly for creating the webview.

Test Plan:
* See video in comment

---
Stack created with [Sapling](https://sapling-scm.com). Best reviewed with [ReviewStack](https://reviewstack.dev/lastmile-ai/aiconfig/pull/1075).
* __->__ #1075
* #1073
* #1070
* #1069
* #1068
* #1066
* #1062
* #1060
* #1059
* #1058
* #1057

### CODE REVIEW

**Comment by saqadri** [comment]
https://github.com/lastmile-ai/aiconfig/assets/25641935/9a6b6405-555d-41a9-b3d6-22dcfe7cfdcd

File: vscode-extension/src/aiConfigEditor.ts [hunk 1]
```diff
@@ -305,10 +306,10 @@ export class AIConfigEditorProvider implements vscode.CustomTextEditorProvider {
     });
 
     // TODO: saqadri - stderr is very noisy for some reason (duplicating INFO logs). Figure out why before enabling this.
-    // startServer.stderr.on("data", (data) => {
-    //   this.extensionOutputChannel.error(this.prependMessage(data, document));
-    //   console.error(`server stderr: ${data}`);
-    // });
+    startServer.stderr.on("data", (data) => {
+      this.extensionOutputChannel.info(this.prependMessage(data, document));
+      console.error(`server stderr: ${data}`);
```
Comment (rholinshead) [inline vscode-extension/src/aiConfigEditor.ts:311]: Based on the comment, should this be enabled yet? Also, info level but logs to error?

**Review by rholinshead — APPROVED** [review]
Accepting to unblock, but see question on the logging

File: vscode-extension/src/aiConfigEditor.ts (hunk 1 above)
Comment (saqadri) [reply vscode-extension/src/aiConfigEditor.ts:311]: Good catch, will fix at top of stack

**Pull request merged**